Project/src/main/java/app/models/Article.java

```diff
@@ -19,8 +19,8 @@
     private String fileType;
     private String review;
     private ArticleState state;
-
     private Date date;
+    private Date reviewDueDate;
```

Comment:
Isn't the 'date' the date the review is due? Or am I missing something?

Reply:
That's the day that it was uploaded.

Reply:
Probably want to change that field to uploadDate